adding cd to server (#439)
CodeForBoulder/upswyng: packages/server/README.md

> You can do this via your system's file explorer or via the following terminal command while in the repo:
> 
> ```console
> cd packages/server

Reviewer: ```suggestion
You can do this via your system's file explorer or via the following terminal command while in the package:

If you're still within the repo's root folder:
`cd packages/server`

Then:
```console
```

We may want to just add this as a caveat to these commands. Otherwise if they're already in the server package, they'll receive a message that the directory doesn't exist.

Author: Great thinking.
